mixer/doc/adapter-development-walkthrough.md

```diff
@@ -423,26 +425,11 @@ Now your adapter is plugged into Mixer and ready to receive data.
 
 To see if your adapter works, we will need a sample operator configuration. So, let's write a simple operator
 configuration that we will give to Mixer for it to dispatch data to your sample adapter. We will need instance, handler
-and rule configuration to be passed to the Mixers configuration server. First we copy a sample attributes config that
-configures Mixer with an attributes vocabulary. We can then use those attributes in the sample operator configuration.
+and rule configuration to be passed to the Mixers configuration server.
 
-Create a directory where we can put sample operator config
-
-```bash
-mkdir sampleoperatorconfig
-```
+Create a sample operator config file with name `mysampleadapter.yaml` inside the `$MIXER_REPO/testdata/config` directory with the following content:
 
-
-Copy the sample attribute vocabulary config
-
-```bash
-cp $MIXER_REPO/testdata/config/attributes.yaml sampleoperatorconfig
-```
-
-
-Create a sample operator config file with name `config.yaml` inside the `sampleoperatorconfig` directory with the following content:
-
-Add the following content to the file `sampleoperatorconfig/config.yaml.`
+Add the following content to the file `$MIXER_REPO/testdata/config/mysampleadapter.yaml.`
```

Comment:
my only concern here is that we will liter `testdata/config` with the adapter configs for all adapters, making it harder to opt out of loading handlers for local testing.

@guptasu thoughts here?